tests/integration/models/reddit/test_live.py

```diff
@@ -98,9 +98,12 @@ def test_invite__already_invited(self, _):
             "TestLiveContributorRelationship_test_invite__already_invited"
         ):
             thread.contributor.invite("nmtake")
-            with pytest.raises(APIException) as excinfo:
+            with pytest.raises(RedditAPIException) as excinfo:
                 thread.contributor.invite("nmtake")
-        assert excinfo.value.error_type == "LIVEUPDATE_ALREADY_CONTRIBUTOR"
```

Comment:
Until we're on PRAW8, this should continue to work, though it should also contain a deprecation warning.

Reply:
It doesn't work like that, because APIException is an alias to the new exception, which doesn't work like the new exception.

Reply:
Right, what I mean is to enable those attributes to work on the first exception in the list on the new class.